Server.java

```diff
@@ -22,9 +22,16 @@ void runServer(ServerUI serverUI) {
 
                 outputToClient.writeBytes("From Server: Connected...\n");
 
+                do {
```

Comment:
This is the logic I had to add, basically I have to wait until the client decides to disconnect